courses/pyladies/plan.yml

```diff
@@ -1,67 +1,99 @@
 - name: Instalace
   materials:
-  - link: lessons/beginners/cmdline
-  - link: lessons/beginners/install
-  - link: lessons/beginners/install_editor
-  - link: lessons/beginners/install_git
+  - name: Úvod do příkazové řádky
```

Comment:
It's complicated due to direct links to pdf files, they don't have their own yaml file, which would define the names, so they need the name specified here. I think it isn't nice to have some items with name and some without it. And another complication was with loading the dictionary of link:name, because pdf files again haven't got a yaml file and it broked the loading.